src/librustc_typeck/check/method/probe.rs

```diff
@@ -192,31 +205,27 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
         // this creates one big transaction so that all type variables etc
         // that we create during the probe process are removed later
         self.probe(|_| {
-            let mut probe_cx = ProbeContext::new(self,
-                                                 span,
-                                                 mode,
-                                                 item_name,
-                                                 steps,
-                                                 opt_simplified_steps);
+            let mut probe_cx =
+                ProbeContext::new(self, span, mode, item_name, steps, opt_simplified_steps);
```

Comment:
This doesn't seem too bad to me, arguably it is better since using less vertical space and the call all on one line is not too hard to read.